cpp/benchmarks/string/case.cpp

```diff
@@ -26,7 +26,7 @@
 void bench_case(nvbench::state& state)
 {
   auto const n_rows    = static_cast<cudf::size_type>(state.get_int64("num_rows"));
-  auto const max_width = static_cast<int32_t>(state.get_int64("width"));
+  auto const max_width = static_cast<int32_t>(state.get_int64("row_width"));
```

Comment:
@davidwendt I slipped in this small fix and wanted to let you know. I saw the benchmarks were failing here, because of a mismatch in axis / parameter names.